### PR TITLE
Theme button

### PR DESCRIPTION
**Pull request by GoodThomas**

### CODE REVIEW

File: src/config/theme.js [hunk 1]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
   ...bodhiTheme.overrides,
   MuiButton: {
     root: {
+      fontSize: xSmallText,
```
Comment (dwalintukan) [inline src/config/theme.js]: should be 1rem, smallText

File: src/config/theme.js [hunk 2]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
+      lineHeight: lineHeight24,
+      padding: `4px ${px(space2X)}`,
```
Comment (dwalintukan) [inline src/config/theme.js]: where did you get these padding values from?

File: src/config/theme.js [hunk 3]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
+      '&:active': {
+        backgroundColor: primaryBrandColor60,
+        boxShadow: 'none',
       },
       '&:disabled': {
-        backgroundColor: primaryBrandColor,
-        color: white,
-        opacity: 0.6,
+        backgroundColor: primaryBrandColor30,
+        color: `${white}!important`,
```
Comment (dwalintukan) [inline src/config/theme.js]: add space in between

File: src/config/theme.js [hunk 4]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
+        boxShadow: 'none',
+      },
+    },
+    outlined: {
+      color: primaryBrandColor,
+      border: `1px solid ${primaryBrandColor}`,
```
Comment (dwalintukan) [inline src/config/theme.js]: it looks like the design has it as 2px?

File: src/config/theme.js [hunk 5]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
+        boxShadow: `1px 1px 4px ${primaryBrandColor60}`,
+      },
+      '&:disabled': {
+        color: primaryBrandColor30,
+        boxShadow: 'none',
       },
     },
     sizeLarge: {
-      fontSize: mediumText,
+      fontSize: smallText,
```
Comment (dwalintukan) [inline src/config/theme.js]: according to the design, large buttons are 24px = `largeText`

File: src/config/theme.js [hunk 6]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
+      lineHeight: lineHeight24,
+      padding: `12px ${px(space2X)}`,
```
Comment (dwalintukan) [inline src/config/theme.js]: where is this padding from?

File: src/config/theme.js [hunk 7]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
       fontWeight: fontWeightBold,
-      minHeight: 56,
+      minWidth: 'auto',
+      minHeight: 'auto',
     },
     sizeSmall: {
       fontSize: xxSmallText,
+      lineHeight: lineHeight16,
+      padding: `4px ${px(space2X)}`,
```
Comment (dwalintukan) [inline src/config/theme.js]: where is the padding from?

File: src/config/theme.js [hunk 8]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
     },
     sizeLarge: {
-      fontSize: mediumText,
+      fontSize: smallText,
+      lineHeight: lineHeight24,
+      padding: `12px ${px(space2X)}`,
       fontWeight: fontWeightBold,
-      minHeight: 56,
+      minWidth: 'auto',
+      minHeight: 'auto',
```
Comment (dwalintukan) [inline src/config/theme.js]: do we need to set min width/height?

File: src/config/theme.js [hunk 9]
```diff
@@ -395,39 +397,88 @@ bodhiTheme.overrides = {
...
     },
     sizeSmall: {
       fontSize: xxSmallText,
```
Comment (dwalintukan) [inline src/config/theme.js]: design says 16px = 1rem

**Comment by GoodThomas** [comment]
It seems to be no `sizeMedium` property, so I overrode styles in `root` property.

File: src/config/theme.js (hunk 8 above)
Comment (GoodThomas) [reply src/config/theme.js]: I set these is to override the default values.

File: src/config/theme.js (hunk 1 above)
Comment (GoodThomas) [reply src/config/theme.js]: It seems to be no `sizeMedium` property, so I overrode styles in `root` property.
Medium size button seems to be 14px.

File: src/config/theme.js (hunk 2 above)
Comment (GoodThomas) [reply src/config/theme.js]: 16px
image
4px = (height 32px - line-height 24px) / 2

File: src/config/theme.js (hunk 6 above)
Comment (GoodThomas) [reply src/config/theme.js]: space3X = 24px
image
12px = (height 48px - line-height 24px) / 2

File: src/config/theme.js (hunk 7 above)
Comment (GoodThomas) [reply src/config/theme.js]: space2X = 16px
image
4px = (height 24px - line-height 16px) / 2

File: src/config/theme.js (hunk 9 above)
Comment (GoodThomas) [reply src/config/theme.js]: image

File: src/config/theme.js (hunk 1 above)
Comment (dwalintukan) [reply src/config/theme.js]: you are right, medium is the default size so root seems like the right spot. and for the font size i was mistaking the line height as the font size... sorry

File: src/config/theme.js (hunk 5 above)
Comment (dwalintukan) [reply src/config/theme.js]: you are right. I mistook line height for font size.

**Pull request merged**